inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/settings/StringMatchingRecommenderTraitsEditor.java

```diff
@@ -95,14 +95,6 @@ protected void onSubmit(AjaxRequestTarget aTarget)
                 && aRecommender.getObject().getId() != null));
         add(uploadField);
     }
-
-    private void actionDeleteGazeteer(AjaxRequestTarget aTarget, Gazeteer aGazeteer)
```

Comment:
We keep the "business logic" of user actions in these `actionXXX` methods to separate logic from UI construction.  It should not be inlined.